docs/csharp/tour-of-csharp/statements.md

```diff
@@ -38,79 +38,79 @@ The `using` statement is used to obtain a resource, execute a statement, and the
 
 The following lists the kinds of statements that can be used, and provides an example for each.
 
-***Local variable declaration***
+Local variable declaration
```

Comment:
Maybe a bulleted list and each sentence with a colon in the end? what do you thing @BillWagner?